plane/source/docs/apms-failsafe-function.rst

```diff
@@ -126,36 +126,72 @@ GCS Failsafe
 **How it works.** When flying while using telemetry on the GCS, the
 autopilot can be programmed to trigger into failsafe mode if it loses
 telemetry. In the event that the autopilot stops receiving MAVlink
-(telemetry protocol) heartbeat messages for more than FS_LONG_TIMEOUT sec, the GCS
-failsafe (FS_GCS_ENABL, 0=Disabled, 1=Enabled) will trigger the
-autopilot to go into long failsafe and change the flight mode to RTL.
+(telemetry protocol) heartbeat messages. :ref:`FS_SHORT_ACTN<FS_SHORT_ACTN>` and :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` apply just in the case of a Throttle Failsafe.
 
 **Setup.**
 
-#. Set FS_GCS_ENABL to 1 to enable it.
+#. Set :ref:`FS_GCS_ENABL<FS_GCS_ENABL>` to 1 to enable it.
 #. Connect to the Mission Planner via telemetry. Verify on the bottom
    right corner of the HUD that you are “flying” in a non auto mode
    (Manual, Stabilize, FBW are ok).
 #. Unplug one of the telemetry radios. After a few minutes power off
-   your autopilot. (Remember the autopilot will not go into failsafe
-   until FS_LONG_TIMEOUT seconds of MAVlink inactivity have passed).
+   your autopilot. (Remember the autopilot will not go fully into failsafe
+   until :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` seconds of MAVlink inactivity have passed).
 #. Connect your autopilot to the mission planner and pull the logs.
-   Verify on the log that the autopilot went into RTL after FS_LONG_TIMEOUT sec of
-   MAVlink inactivity.
+   Verify on the log that the autopilot went into RTL after :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` sec of MAVlink inactivity.
+
+Battery Failsafe
+~~~~~~~~~~~~~~~~
+
+.. note::
+
+    This failsafe requires the vehicle have a working :ref:`Power Module <common-powermodule-landingpage>`.
+
+.. note:: ArduPilot firmware versions 4.0 and later support up to 10 batteries/power monitors. All the  discussion below applies to those optional batteries also. Each can trigger a failsafe and each can have different actions and setup values. In addition, a group of batteries can be treated as a single unit, see ``BATTx_MONITOR`` = 10.
+
+When the failsafe will trigger
+------------------------------
+
+If enabled and set-up correctly the battery failsafe will trigger if the main battery's
+
+-  voltage drops below the voltage held in the :ref:`BATT_LOW_VOLT <BATT_LOW_VOLT>` parameter (or FS_BATT_VOLTAGE in older versions) for more than 10 seconds.  The default voltage is 10.5 volts.  If set to zero the voltage based trigger will be disabled.
```

Comment:
This looks wrong. Default is 0V on plane. It's actually only 10.5V on copter.

Reply:
fixed